restler/engine/transport_layer/messaging.py

```diff
@@ -53,12 +53,15 @@ def __init__(self, connection_settings):
                     context = ssl._create_unverified_context()
                 else:
                     context = ssl.create_default_context()
-                if Settings().client_certificate_path:
+                if Settings().client_certificate_path and Settings().client_certificate_key_path:
                     context.load_cert_chain(
```

Comment:
If you add this 'and', the code path without a key file is broken.  I believe if you just remove the 'and', the previous code path will continue to work.